secc-daemon.js

```diff
@@ -154,9 +161,6 @@ if (cluster.isWorker) {
     }
   }
 
-  var schedulerUrl = 'http://' + SECC.daemon.scheduler.address + ':' + SECC.daemon.scheduler.port;
-  var socket = require('socket.io-client')(schedulerUrl);
```

Comment:
'socket' is widely used in cluster slaves.
you need to implement other alternative logics in order to remove 'socket'.

ex) many exceptions raised.
     secc/routes/daemonCompilePreprocessed.js
 -- if (socket.connected && jobId) socket.emit('compileBefore', { jobId: jobId });
    DAEMON.worker.emit('compileBefore', { jobId: jobId });

    secc/routes/daemonCache.js 
    if (socket.connected) socket.emit('cacheNotExists', metaData);